wheesko/PSKProject
# Register ui

To test happy path:
- Login into with admin user;
- Get JWT;
- Create a worker with provided email;
- Logout;
- Login with new worker email and password;
- Input name, surname, password;
- Login once more;
Expected result: main app is visible.

**EXTRA:**
Password is matching regex using one lowercase char, uppercase, number, symbol of a certain array (provided in UI)
